packages/forklift-console-plugin/src/mock-console-extension/setupBrowserWorker.tsx

```diff
@@ -0,0 +1,46 @@
+import { type SetupWorker, rest, setupWorker } from 'msw';
+
+import { allHandlers } from '@kubev2v/mocks/console-handlers';
+
+export { type SetupWorker } from 'msw';
+
+const fancyLog = (url, ...rest) => [
+  '%cmock-plugin passthrough%c \u{1fa83} %s',
+  'font-weight: bold',
+  'font-weight: normal',
+  url,
+  ...rest,
+];
+
+/**
+ * Setup the MockServiceWorker with all of the mock handlers available handling request
+ * for UI url paths.  The worker will be added to `window.msw` for debugging convenience.
+ *
+ * @param staticFilePath URL prefix where `mockServiceWorker.js` will be available
+ * @returns A setup and started MockServiceWorker
+ */
+export function setupBrowserWorker(staticFilePath: string): SetupWorker {
+  const worker = setupWorker(
+    ...allHandlers,
+
+    rest.all('/*', (req) => {
+      console.log(...fancyLog(req.url));
```

Comment:
console log in the middle of the code is problematic, can you wrap it in some helper/util log method that will do the actual console.log

Reply:
Putting `console.log()` in another place breaks the view in source link functionality on the browser console.  I keep it here so clicking the link takes me directly to its location in source.

Reply:
that is a price I'm willing to pay, IMHO console log is usefully even without direct link

Reply:
Check current version.

The handler with this console.log() is simply an intercept, log, proxy handler to show that MSW is seeing a URL.  It is useful just to show URLs that are not being handled by any other handlers.

As for a console.log in the middle of code being problematic ... it is code to help developers.  I don't see any problems with that.